api_app/db/repositories/resources.py

```diff
@@ -100,7 +100,7 @@ async def validate_input_against_template(self, template_name: str, resource_inp
 
         return parse_obj_as(ResourceTemplate, template)
 
-    async def patch_resource(self, resource: Resource, resource_patch: ResourcePatch, resource_template: ResourceTemplate, etag: str, resource_template_repo: ResourceTemplateRepository, resource_history_repo: ResourceHistoryRepository, user: User, force_version_update: bool = False) -> Tuple[Resource, ResourceTemplate]:
+    async def patch_resource(self, resource: Resource, resource_patch: ResourcePatch, resource_template: ResourceTemplate, etag: str, resource_template_repo: ResourceTemplateRepository, resource_history_repo: ResourceHistoryRepository, user: User, force_version_update: bool = False, resource_action: str = 'upgrade') -> Tuple[Resource, ResourceTemplate]:
```

Comment:
Are there cases where resource_action won't be passed?
Should we just always pass it?

Reply:
Done

Reply:
Done